WordPress/src/androidTest/java/org/wordpress/android/e2e/EditorTests.java

```diff
@@ -105,4 +105,25 @@ public void testPublishFullPost() {
         boolean isPublished = editorPage.publishPost();
         assertTrue(isPublished);
     }
+
+    @Test
+    public void testPublishSimplePage() {
```

Comment:
This test involves publishing a simple page? Was that intended to be a part of this PR, from what I understood this PR was only to contain the test for the activity log?